src/main/app/claims/claimModelConverter.ts

```diff
@@ -69,7 +69,7 @@ export class ClaimModelConverter {
           individualDetails.name,
           this.convertAddress(individualDetails.address),
           individualDetails.hasCorrespondenceAddress ? this.convertAddress(individualDetails.correspondenceAddress) : undefined,
-          draftClaim.claimant.mobilePhone.number,
+          draftClaim.claimant.phone.number,
```

Comment:
can this be done in two stages please? have the converter check for the old version and use it is it's there and if not then use the new one. As the phone number is optional if we do this all the phone numbers in the drafts will be lost and the users won't notice.

Reply:
Agreed, and noting that I had asked you about this previously and had confirmation to go ahead regardless.

Reply:
6d6454ad5a1afe432338388f8349b27fe46d88ec

Reply:
I must have been having a stressful day and hadn't considered the fact that it was optional. if it was a mandatory field then the users would be notified and be able to put it in after.